coder_sniffer/Drupal/Sniffs/InfoFiles/DependencyArraySniff.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Largely copied from
+ * PHP_CodeSniffer\Standards\Squiz\Sniffs\Classes\ClassFileNameSniff.
+ *
+ * Extended to support anonymous classes and Drupal core version.
+ */
+namespace Drupal\Sniffs\InfoFiles;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Dependency should have single entry as an array in Drupal 8 info files.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
+ */
+class DependencyArraySniff implements Sniff
+{
+
+
+    /**
+     * Returns an array of tokens this test wants to listen for.
+     *
+     * @return arrayint|string
+     *   Returns array or string.
+     */
+    public function register()
+    {
+        return [T_INLINE_HTML];
+
+    }//end register()
+
+
+    /**
+     * Processes this test, when one of its tokens is encountered.
+     *
+     * @param \PHP_CodeSniffer\Files\File $phpcsFile The file being scanned.
+     * @param int                         $stackPtr  The position of the current token in the stack passed in $tokens.
+     *
+     * @return int Returns integer.
```

Comment:
the comment just repeats what `@return int` says, so can be removed. You can leave that out as we do on other sniff classes.

Reply:
sure